frontend/registration.html

```diff
@@ -58,10 +58,12 @@ <h3 class="card-title">Registration</h3>
                 <input type="password" class="form-control" id="confirm_password" />
               </div>
               <button type="submit" class="btn btn-primary" value="Submit">Submit</button>
-              <small class="form-text text-muted"
-                >Already have an account? Login
-                <a href="./index.html">here</a></small
-              >
+			  
+			  <!-- link to bio_registration.html, little fingerprint icon to indicate biometric registration-->
+              <a href = "bio_registration.html"><img src = "assets/images/fingerprint.png" height="42" width="42" style="float: right;"></a>
```

Comment:
Same thing about moving styles into the CSS file. Another thing that you might not know: if the image has an aspect ratio of 1x1, you only have to set the `height` or the `width`, and the other property will match that value. This is because the default styling property value for `height` and `width` is `auto`, which preserves the original aspect ratio.